### PR TITLE
Coding challenge frontend react volodymyrl

### PR DESCRIPTION
**Pull request by volodymyrl**

### CODE REVIEW

**Comment by favorit** [comment]
One of the tasks was to make that app public. Are you interesting to make it? It could help with:
- checking if it actually works
- understanding of your experience with deployment tools (Operations)

**Comment by volodymyrl** [comment]
Hi Dmitriy,

I've deployed this app to heroku and add link to README file
*https://github.com/volodymyrl/coding-challenge-frontend-react#demo
<https://github.com/volodymyrl/coding-challenge-frontend-react#demo>*

Best,
Volodymyr

File: src/components/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import Pagination from "../Pagination";
+import { Container } from "../ui/Layout/StyledLayout";
+import Text from "../ui/Text";
+import IncidentsList from "../IncidentsList";
+import { incidentPropTypes } from "../../constants/propTypes";
+import ErrorMessage from "../ui/ErrorMessage";
+import ErrorBoundary from "../ErrorBoundary";
+
+class HomePage extends Component {
+  componentDidMount = () => {
```
Comment (favorit) [inline src/components/HomePage/HomePage.jsx]: why do you need to define `componentDidMount` in such way?

```suggestion
  componentDidMount() {
```

**Comment by favorit** [comment]
Few moments to discuss during tech interview:
1. typescript
2. quality of tests: enzyme shallow snapshots against `react-testing-library` approach
3. prettier and linter

File: src/components/HomePage/HomePage.jsx (hunk 1 above)
Comment (volodymyrl) [reply src/components/HomePage/HomePage.jsx]: No need to use the arrow function there,  I just use a vscode snippets package for autocomplete React/Redux code and it has the arrow function in the componentDidMount snippet

**Comment by volodymyrl** [comment]
Sounds good, when we can have a tech interview?

**Comment by favorit** [comment]
Tomorrow Gui (our PO) coming back and he will communicate with you and Eugene about next steps. 
But personally, I guess we will have the tech interview this week.

Stay tuned!

**Comment by volodymyrl** [comment]
Thanks for updates! Have a good day!

**Pull request closed without merging**